tests/configurations/integration/jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  transform: {
+    '^.+\\.ts$': 'ts-jest',
+  },
+  coverageReporters: ['text', 'html'],
+  rootDir: '../../../.',
+  testMatch: ['<rootDir>/tests/integration/**/*.spec.ts'],
+  setupFiles: ['<rootDir>/tests/configurations/jest.setup.js'],
+  reporters: [
+    'default',
+    [
+      'jest-html-reporters',
+      { multipleReportsUnitePath: './report', pageTitle: 'integration', publicPath: './reports', filename: 'integration.html' },
```

Comment:
```suggestion
      { multipleReportsUnitePath: './reports', pageTitle: 'integration', publicPath: './reports', filename: 'integration.html' },
```
should match the unit config too.

Reply:
done